py/farm_ng/canbus/packet.py

```diff
@@ -491,3 +492,133 @@ def is_button_pressed(self, button: PendantButtons) -> bool:
 
     def __str__(self):
         return "x {:0.3f} y {:0.3f} buttons {}".format(self.x, self.y, self.buttons)
+
+
+class BugDispenserRpdo3(Packet):
```

Comment:
I would call it BugDispenserCommand. It just happens to be sent on PDO3 channels w/ dashboard node ID

```suggestion
class BugDispenserCommand(Packet):
```